### DISCUSSION TITLE
Can I make people who enter in my room move something in there?

### DISCUSSION THREAD

**Discussion by Hong0306** [mozilla/hubs#5794] [Spoke Questions] [answered]
Hi!
I want to make a tree that avatars can pick fruit from.
Is this possible?
If so, how can I make it?

**Comment by matthewbcool** [comment, marked as answer]
One way you can try is with the spawner component. This will provide you with endless fruit. Another way is by using the 'pin' feature and then allowing logged-in participants to unpin. 
 
<img width="1406" alt="image" src="https://user-images.githubusercontent.com/4493657/201968294-9cce7781-b864-434f-b8ff-1d1ec4b98de0.png">


https://user-images.githubusercontent.com/4493657/201969449-f7c514ea-4e0f-4f2e-afdf-3692546e15b2.mp4

**Comment by Hong0306** [comment]
Thank you!!
